src/PowerPlaywright.Strategies/Controls/Pcf/PcfGridControl.cs

```diff
@@ -45,6 +46,7 @@ public PcfGridControl(IAppPage appPage, string name, IEnvironmentInfoProvider in
             this.treeGrid = this.Container.GetByRole(AriaRole.Treegrid);
             this.rowsContainer = this.Container.Locator("div.ag-center-cols-viewport");
             this.columnHeaders = this.Container.Locator("[role='columnheader']:not([aria-colindex='1'])");
+            this.gridHeaderContainer = this.Container.Locator("div[class='ag-header-container']");
```

Comment:
Prefer to go for a more accessibility-based selector here. To get the row group that has column headers, something like this would work:

```csharp
this.Container.GetByRole(AriaRole.Rowgroup).Filter(new LocatorFilterOptions { Has = this.Page.GetByRole(AriaRole.Columnheader) });
```